skatepark/sass/blocks/_post-comments.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../../blockbase/sass/blocks/button-mixins";
+
+.wp-block-post-comments {
+	.reply {
+		a {
+			--wp--custom--button--typography--font-size: var(--wp--preset--font-size--normal);
+			@include button-border-styles;
+			--wp--custom--button--color--text: var(--wp--custom--button--hover--color--text);
+			--wp--custom--button--color--background: var(--wp--custom--button--color--background);
+			--wp--custom--button--border--color: var(--wp--custom--button--hover--color--text);
+
+			& ,
+			&:hover,
+			&:focus,
+			&:active {
+				line-height: 1;
+			}
+			&:not(.has-background):not(.has-text-color) {
+				&:hover,
+				&:focus,
+				&.has-focus {
+					--wp--custom--button--typography--font-size: var(--wp--preset--font-size--normal);
+					--wp--custom--button--color--text: var(--wp--custom--button--hover--color--background);
+					--wp--custom--button--color--background: var(--wp--custom--button--hover--color--text);
+				}
+			}
+		}
+	}
+
+	.comment-reply-title {
+		margin-bottom: 30px;
+	}
+
+	form {
+		display: grid;
+		column-gap: 1.2em;
+		grid-template-rows: auto;
+		grid-template-areas:
+			"notes notes"
+			"comment comment"
+			"author email"
+			"url url"
+			"cookies-consent cookies-consent"
+			"form-submit form-submit";
```

Comment:
I just noticed that this is repeated on Quadrat. I think it's safe to handle this in Blockbase too and repeat less code over the child themes, since they are mostly going to have the same layout, what do you think @kjellr?

Reply:
Yeah, definitely. I expect this comment form layout will be mostly identical across of our themes.